redbot/cogs/reports/reports.py

```diff
@@ -262,15 +262,15 @@ async def report(self, ctx: commands.Context, *, _report: str = ""):
             return await author.send(
                 _(
                     "You've sent too many reports recently. "
-                    "Please contact a server admin if this is important matter, "
+                    "Please contact a server admin if this is an important matter, "
                     "or please wait and try again later."
                 )
             )
         if author.id in self.user_cache:
             return await author.send(
                 _(
-                    "Please finish making your prior report before trying to make an "
-                    "additional one!"
+                    "Please finish making your prior report before trying to make a "
+                    "new one!"
```

Comment:
Make this a one line and it will work
`"Please finish making your prior report before trying to make a new one!"`